frontend/tests/unit/store.shoots.spec.js

```diff
@@ -118,6 +129,12 @@ describe('store.shoots.getters', () => {
         }
       }
     ]
+
+    state = { shoots: fromPairs(map(shootItems, shootItem => [keyForShoot(shootItem.metadata), shootItem])), filteredShoots: shootItems, freezeSorting: false, freezedStaleShoots: [] }
```

Comment:
```suggestion
    state = {
      shoots: fromPairs(map(shootItems, shootItem => [keyForShoot(shootItem.metadata), shootItem])),
      filteredShoots: shootItems,
      freezeSorting: false,
      freezedStaleShoots: []
    }
```
added newlines for better readability